pages/common/adb-install.md

```diff
@@ -15,6 +15,10 @@
 
 `adb install -r {{path/to/file.apk}}`
 
+- Allow version code downgrade (debuggable packages only):
```

Comment:
```suggestion
- Enable the function to use previous code (debuggable packages only):
```

I don't really know if this is correct, what do you think @HUGHNew

Reply:
```suggestion
- Push an Android application allowing version code downgrade (debuggable packages only):
```

Reply:
I choose the better one in my opinion.